client/components/FormattingBar/positioning.js

```diff
@@ -0,0 +1,39 @@
+import { Selection, NodeSelection } from 'prosemirror-state';
```

Comment:
I've preferred keeping all prosemirror imports within the `@pubpub/editor` package. For use as we have below, I'd typically create a helper function in the editor package and `import { createNodeSelection } from '@pubpub/editor'`. I can understand that feeling verbose, but it does keep us from having different prosemirror package versions within the code base. 

What are your thoughts on that layer of abstraction?

Reply:
Overall I'd like to find a way to write Prosemirror-oriented code inside of `pubpub` when it doesn't feel like the code is core to the functionality of `@pubpub/editor`. I hadn't considered the version matching thing, which is interesting — it seems like what I really want is a way to enforce that `pubpub` grabs the same version of all the Prosemirror packages as `@pubpub/editor`. For the time being, I'll move this to the editor package.